x-pack/plugins/entity_manager/server/routes/v2/search.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import moment from 'moment';
+import { z } from '@kbn/zod';
+import { createEntityManagerServerRoute } from '../create_entity_manager_server_route';
+import { entitySourceSchema } from '../../lib/queries';
+import { UnknownEntityType } from '../../lib/entities/errors/unknown_entity_type';
+
+export const searchEntitiesRoute = createEntityManagerServerRoute({
```

Comment:
We didn't include any OAS annotation for this but maybe we ignore that for the v2 things for the time being since they're so likely to change? And they're still internal APIs, WDYT?

Reply:
sgtm